docs/reference/tab-widgets/troubleshooting/data/restore-from-snapshot.asciidoc

```diff
@@ -0,0 +1,212 @@
+. Use the left side menu to go to **Snapshot and Restore**, there you can verify that you have a snapshot to restore
+the target indices and data streams from. By clicking on every snapshot you can see the indices and data streams it
+contains. Not all resources need to be restored from the same snapshot. **Do not proceed** if there is no backup of the
+affected resources in a snapshot that is up-to-date enough for your use case, otherwise this will result in further data
+loss.
++
+[role="screenshot"]
+image::images/troubleshooting/data/kibana-restore-snapshot-content.png[{kib} snapshots,align="center"]
++
+. Go back to **Index Management** via the left side menu. In **Indices** enable **Include hidden indices** toggle and
+delete all the indices that have noted, by selecting them and then using the **Manage index** button
+choose delete index. In the **Data Streams** tab repeat the same by selecting the `red` data streams and then
+clicking **Delete data steam**.
++
+. Use the left side menu to go back to **Snapshot and Restore**, click on the snapshot that you would like to restore
+from then click **Restore**. In the next page disable the toggle *All data streams and indices, including system
+indices*, deselect all and then choose the indices and data streams you would like to restore.
++
+[role="screenshot"]
+image::images/troubleshooting/data/kibana-restore-snapshot-select-resources.png[{kib} select resources to restore,align="center"]
++
+. Click **Next** twice to continue without changing anything else and finally click **Restore Snapshot**.
++
+[role="screenshot"]
+image::images/troubleshooting/data/kibana-restore-snapshot-confirm.png[{kib} confirm snapshot restore,width=30%]
++
+. When the process will be complete you will see in the **Restore Status** tab the result of your restore.
++
+[role="screenshot"]
+image::images/troubleshooting/data/kibana-restore-snapshot-result.png[{kib} select resources to restore,align="center"]
++
+For more guidance on creating and restoring snapshots see
+<<snapshot-restore, this guide>>.
+
+//end::kibana-api-ex[]
+// end::cloud[]
+
+// tag::self-managed[]
+In order to restore the indices that are missing shards:
+
+. View the affected indices shards using the <<cat-indices,cat indices API>>.
++
+[source,console]
+----
+GET _cat/indices?v&health=red&h=index,status,health
+----
++
+The response will look like this:
++
+[source,console-result]
+----
+index                                status health
+.ds-my-data-stream-2022.06.17-000001 open   red
+kibana_sample_data_flights           open   red
+----
+// TEST[skip:illustration purposes only]
++
+The `red` health of the indices above indicates that these indices are missing primary shards,
+meaning they are missing data.
++
+. In order to restore the data we need to find a snapshot that contains these two indices.To find
+such a snapshot use the
+<<get-snapshot-api,get snapshot API>>.
++
+[source,console]
+----
+GET _snapshot/snap-repo/*?verbose=false
```

Comment:
```suggestion
GET _snapshot/my_repository/*?verbose=false
```
Keep it consistent with the restore-from-snapshot.html doc, also without the word `my`, it's hard for users to distinguish variables from ES API commands.

Reply:
Good point, I will fix all occurrences.